docs/reference/cluster_manifest.md

```diff
@@ -144,6 +143,14 @@ These parameters are grouped directly under  the `spec` key in the manifest.
   [the reference schedule format](https://kubernetes.io/docs/tasks/job/automated-tasks-with-cron-jobs/#schedule)
   into account. Optional. Default is: "30 00 \* \* \*"
 
+* **additionalVolumes**
+  List of additional volumes to mount in each container of the statefulset pod.
+  Each item must contain a `name`, `mountPath` and `volumeSource` which is a
+  [kubernetes volumeSource](https://godoc.org/k8s.io/api/core/v1#VolumeSource).
+  It allow you to mount existing PersistentVolumeClaim, ConfigMap, Secrets inside the Statefulset.
```

Comment:
- "It allow**s** you"
- Either all plural or singular for the types.
- casing Statefulset -> StatefulSet.

Suggestion:

```
It allows you to mount existing PersistentVolumeClaims, ConfigMaps and Secrets inside the StatefulSet.
```

Reply:
fixed. Sorry for my bad english (I'm french)